### ISSUE TITLE
Feature request: add variant ontology term for variants in RNU4-2

### ISSUE THREAD

**Issue by berntpopp** [berntpopp/sysndd#66]
The frequent variant in RNU4-2 (https://www.medrxiv.org/content/10.1101/2024.04.07.24305438v2.full.pdf) has currently no term.
Add an appropriate term for this variant to the ontology list.

Table of ontology terms used can be found here:
https://github.com/berntpopp/sysndd/blob/master/documentation/data/lists_2022-05-03.xlsx

**Comment by simonfruh** [comment]
We updated variant ontology list with terms for variants in non-coding RNA genes.
[lists_2022-05-03 terms used from 20240515.xlsx](https://github.com/berntpopp/sysndd/files/15321923/lists_2022-05-03.terms.used.from.20240515.xlsx)

**Comment by berntpopp** [comment]
Fixed using SQL statement according to new list:

```
UPDATE sysndd_db.variation_ontology_list
SET is_active = 1
WHERE vario_id IN ("VariO:0350", "VariO:0382", "VariO:0400", "VariO:0401", "VariO:0468", "VariO:0478", "VariO:0485", "VariO:0492", "VariO:0493", "VariO:0494", "VariO:0495");
```

closes #66
